templates/blog/post_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+<h1>{{ post.title }}</h1>
+<p>{{ post.content }}</p>
+<h3>{{ post.commentary_count }} comment{{ post.commentary_count|pluralize }}:</h3>
```

Comment:
The variable `post.commentary_count` is used here. Make sure that this variable is defined in the context or as a property in the model to avoid any undefined variable errors.